src/babelRegister.js

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
```

Comment:
Do we need this shebang? Do we ever execute this file directly?

Reply:
Nope